### PR TITLE
Issue 104 - Add facebook stupid counter fix widget to sum HTTP and HTTPS shares of

### LINKED ISSUE #104: Migration to HTTPS zeroes Facebook page likes counters
Facebook differentiates URIs with different schema; technically that's correct, but in practice is a very stupid assumption that leads to the reset of all page likes counters on a site after the migration from HTTP to HTTPS.

this issue is very well documented since many years and Facebook has done nothing to solve or even mitigate it.

there are 2 known workarounds, according to the [developer's documentation](https://developers.facebook.com/docs/sharing/webmasters/crawler):

* exempt the Facebook crawler from the HTTPS redirect
* use the old page as the canonical URL for the new page

both of them could probably lead to lost of like counters of some other pages.

### PR DESCRIPTION
**Pull request by rodfersou**
same page

closes #104

### CODE REVIEW

**Comment by rodfersou** [comment]
@hvelarde can you help me with this failiing test for Plone 5.1 ?  I don't know what is going on here..

**Comment by hvelarde** [comment]
please remove the commit with the translations as they are completely wrong, at least in Spanish.

File: sc/social/like/plugins/facebook/templates/plugin.pt [hunk 1]
```diff
@@ -16,8 +16,7 @@
              tal:attributes="data-layout view/typebutton;
                              data-href view/url;"></div>
         <div class="fb-stupid-counter-widget"
-            tal:condition="view/fbstupid_counter_fix">
-            <span i18n:translate="" tal:omit-tag="">Share</span> <span class="count">0</span>
-        </div>
+            tal:condition="view/fbstupid_counter_fix"
+             i18n:translate>Share <span class="count">0</span></div>
```
Comment (rodfersou) [inline sc/social/like/plugins/facebook/templates/plugin.pt]: the page template changes should not be reverted..

**Comment by rodfersou** [comment]
Will be implemented in a different way see https://github.com/collective/sc.social.like/issues/104#issuecomment-309073490

**Pull request closed without merging**